.github/workflows/Lint_Build.yaml

```diff
@@ -43,10 +43,6 @@ jobs:
           echo "${REVIVE_OUTPUT}"
           exit 1
         fi
-    - name: Build
-      run: go build -v -buildmode=plugin -o mexport.so
-    - name: Build Docker Image
```

Comment:
Will this still be built as part of the CI running for PRs?

Reply:
No. Do you prefer it to run a test build?